src/HIE/Bios/Types.hs

```diff
@@ -116,11 +120,31 @@ instance Pretty Log where
           ]
     where
       envText = map (indent 2 . pretty) $ prettyProcessEnv cp
+  pretty (LogRequestedCradleLoadStyle crdlName ls) =
+    "Request loading" <+> pretty crdlName <+> "cradle using" <+> case ls of
+      SingleComponent -> "Single Component Strategy"
+      LoadWithContext fps -> "Multiple Components Strategy:" <> line <> indent 4 (pretty fps)
```

Comment:
It seems like from the `hie-bios` perspective the truth is that it's either one _file_ or _many files_, right? `hie-bios` doesn't particularly judge whether or not they're in different components or anything. Just that you can ask for flags to load one file, or a bunch of files.

Reply:
Yeah, that is true. But logs are still kind of user facing, so I thought, it might be better to use the terminology we would use in HLS.

Should we uniformly use the `hie-bios` perspective here?

Reply:
I think it makes `hie-bios` more confusing if we're not consistent. But maybe we're just a bit confused about what `hie-bios` is saying it will do? Or perhaps we can make the message informative enough to cover both: "Requested to load cradle using all open files (multi-component)" or something?

Reply:
that's a good suggestion, updated!

Reply:
I also renamed the `LoadStyle` constructors to reflect this more accurately.